src/gpd_payments.tf

```diff
@@ -0,0 +1,76 @@
+# Subnet to host the api config
+module "payments_snet" {
+  source                                         = "git::https://github.com/pagopa/azurerm.git//subnet?ref=v1.0.51"
+  name                                           = format("%s-payments-snet", local.project)
+  address_prefixes                               = var.cidr_subnet_payments
+  resource_group_name                            = azurerm_resource_group.rg_vnet.name
+  virtual_network_name                           = module.vnet.name
+  enforce_private_link_endpoint_network_policies = true
+
+  delegation = {
+    name = "default"
+    service_delegation = {
+      name    = "Microsoft.Web/serverFarms"
+      actions = ["Microsoft.Network/virtualNetworks/subnets/action"]
+    }
+  }
+}
+
+
+
+module "payments_app_service" {
+  source = "git::https://github.com/pagopa/azurerm.git//app_service?ref=v2.2.0"
+
+  resource_group_name = azurerm_resource_group.gpd_rg.name
+  plan_type           = "external"
+  plan_id             = azurerm_app_service_plan.gpd_service_plan.id
```

Comment:
@pasqualedevita we using this `azurerm_app_service_plan.gpd_service_plan` for several things , in details : 
- for 2 app service
- and 3 fn app 
do you see possible problems in this?